tex/main/MarkovChain/closed_form_state_probs/main.tex

```diff
@@ -1,15 +1,28 @@
 \subsection{Graph Theoretical Approach}
+% TODO: Generalise notation in entire section 
 
-This section aims to investigate the connection between Markov chains and graph theory by exploring alternative ways of calculating the state probabilities and considering the problem from different perspectives.
+This section aims to investigate the connection between Markov chains and graph theory by exploring alternative ways of calculating the steady state probabilities and considering the problem from different perspectives.
 
 \subsubsection{Parameters}
-The inputs of the formula are the number of servers \(C\), the threshold \(T\), the system capacity \(N\) and the parking capacity \(M\). 
```

Comment:
You missed the first sentence in my suggestion.

Reply:
Oops. My bad